docs/modules/plugins/pages/plugin-mobitru.adoc

```diff
@@ -43,6 +58,41 @@ selenium.grid.capabilities.platformVersion=12
 selenium.grid.capabilities.deviceName=iPhone 13
 ----
 
+IMPORTANT: For search more specific device you can additionally use
+mobitru device search capabilities specified below. The `deviceName` and `udid` capabilities are prioritized over mobitru device search capabilities when searching for a device.
+
+NOTE: The properties marked with *bold* are mandatory.
+
+[#mobitru_device_search_capabilities]
```

Comment:
I think this section should be move to the part describing the process of device taking